config/config_loader.py

```
@@ -56,6 +57,22 @@ def matches(self, client_ip) -> bool:
+    # type: (str, str, str) -> List[float]
+    tokens = [t.strip() for t in value.split(",")]
+    result = []
+    for token in tokens:
+        if not token:
+            continue
+        try:
+            result.append(float(token))
+        except ValueError:
+            raise ValueError(
+                f"Invalid {key_name} value '{token}' in section [{section}]"
+            )
```

Ruff (0.15.6)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)